exporter/prometheus/prometheus_test.go

```diff
@@ -343,6 +343,93 @@ func TestCumulativenessFromHistograms(t *testing.T) {
 	}
 }
 
+func TestHistogramUnorderedBucketBounds(t *testing.T) {
+	exporter, err := NewExporter(Options{})
+	if err != nil {
+		t.Fatalf("failed to create prometheus exporter: %v", err)
+	}
+	view.RegisterExporter(exporter)
+	reportPeriod := time.Millisecond
+	view.SetReportingPeriod(reportPeriod)
+
+	m := stats.Float64("tests/bills", "payments by denomination", stats.UnitDimensionless)
+	v := &view.View{
+		Name:        "cash/register",
+		Description: "this is a test",
+		Measure:     m,
+
+		// Intentionally used unordered elements in the ascending distribution
+		// to ensure unordered bucket bounds are handled.
+		Aggregation: view.Distribution(10, 5, 1, 50, 20, 100, 250),
+	}
+
+	if err := view.Register(v); err != nil {
+		t.Fatalf("Register error: %v", err)
+	}
+	defer view.Unregister(v)
+
+	// Give the reporter ample time to process registration
+	<-time.After(10 * reportPeriod)
+
+	values := []float64{0.25, 245.67, 12, 1.45, 199.9, 7.69, 187.12}
+	// We want the results that look like this:
+	// 1:   [0.25]      		| 1 + prev(i) = 1 + 0 = 1
+	// 5:   [1.45]			| 1 + prev(i) = 1 + 1 = 2
+	// 10:	[]			| 1 + prev(i) = 1 + 2 = 3
```

Comment:
There is a problem here, 7.69 has to be in 10 bucket. Or is that just a bad copy+paste?

Reply:
I copied and pasted it from https://github.com/census-instrumentation/opencensus-go/blob/ed9f155754b50f84f15997a4550ebd54667078e7/exporter/prometheus/prometheus_test.go#L288-L295. Looks like the previous comments are wrong. Updated.

Reply:
I think they got stale during the removal of 0 as the first bucket bound, but now this is cool. Thanks for fixing them!